### ISSUE TITLE
Copy site fails with multiple errors on log  `ERROR: duplicate key value violates unique constraint "unique_workflow_task"`

### ISSUE THREAD

**Issue by fabrizzio-dotCMS** [dotCMS/core#17483]
### Describe the bug
Site Copy Fails with multiple errors on the log like: 
ERROR: duplicate key value violates unique constraint "unique_workflow_task"
Evidently this is caused by the recent introduction of a unique constraint that is meant to enforce the uniqueness of workflow tasks per language and web-asset.

### Steps to reproduce the behavior:
1. Go to System/Sites
2. Click on `Add new site` and Select the option  `Copy an existing Site`
3. Hit Next and Make sure you select `Copy All` Hit Next again
4. Set a `Host Name` and Hit `Save Activate`
5. You'll see that the site is partially copied and a bunch of errors `duplicate key value violates unique constraint "unique_workflow_task"` pop-up in the logs.

### Expected behavior
The site should be copied completely. Now it doesn't

**Comment by fabrizzio-dotCMS** [comment]
Related to #17088

**Comment by jgambarios** [comment]
Recently we found during tests:

```
db.DotConnect - Executing ALTER TABLE workflow_task ADD CONSTRAINT unique_workflow_task UNIQUE (webasset,language_id)
10:49:58.115  ERROR db.HibernateUtil - ---------- DotHibernate: error on rollbackTransaction ---------------
org.postgresql.util.PSQLException: Cannot rollback when autoCommit is enabled.
com.dotmarketing.exception.DotDataException: ERROR: could not create unique index "unique_workflow_task"
  Detail: Key (webasset, language_id)=(72839e70-7f8e-412f-a895-fd17aa920e14, 1) is duplicated.
```

And the problem is because before running the upgrade task we already had duplicate data in the `workflow_task` table.

We need to analyze what options do we have, clean up the data before to apply the constraint?, on fail just continue without adding the constraint?.

One option could be to use the `HAVING` clausule, something like:

```
GROUP BY webasset
 HAVING COUNT(*)>1;
```

We need to see how heavy could be using the `HAVING` in a big dataset.

**Comment by bryanboza** [comment]
Please test doing a copy of the index page under the  /about-us/ folder, right now we are getting an error and we are unable to copy the page: 
Error: https://gist.github.com/bryanboza/c1879b8454d3f0b63e396956269b9efc 
SC: https://www.screencast.com/t/APwFfTFY

**Comment by dsilvam** [comment]
Needs Work: Copied host not including content. Right after copying a host from demo.dotcms.com, check `about-us/index` for example and you will see no content.

**Comment by fabrizzio-dotCMS** [comment]
This problem is also present in demo.dotCMS.com it is clearly a separate bug.

**Comment by fabrizzio-dotCMS** [comment]
The issue is evident only when copying a whole site. A simple Page Copy works correctly.

**Comment by jgambarios** [comment]
Found issue with the content of pages will be fix in https://github.com/dotCMS/core/issues/17541

**Comment by bryanboza** [comment]
Fixed, tested on the release-5.2.1 branch and I'm unable to reproduce the main problem, a new related issue reported in a new card.

**Comment by bryanboza** [comment]
Fixed, tested on master // Postgres // FF
